Gemfile

```diff
@@ -61,8 +61,8 @@ group :frontend do
 end
 
 group :backend do
-  gem 'teaspoon', github: 'jejacks0n/teaspoon', require: false
-  gem 'teaspoon-mocha', github: 'jejacks0n/teaspoon', require: false
+  gem 'teaspoon', require: false
```

Comment:
What about locking it to `~> 1.2` on this branch? It seems like the next major will require Ruby 3.

Reply:
Makes sense. Updated